src/test/java/myfirstproject/service/impl/OperationsTests.java

```diff
@@ -46,53 +47,58 @@ public void testInitializedOperations_ok() {
         operation.put(Operation.PURCHASE.getOperation(), PURCHASE_OPERATION);
         operation.put(Operation.RETURN.getOperation(), RETURN_OPERATION);
         SEPARATION.toDoEachOperation(FRUIT_DAO, TEMPORAL_LIST, operation);
+        Assert.assertEquals(Operation.BALANCE.getOperation(), "b");
+        Assert.assertEquals(Operation.SUPPLY.getOperation(), "s");
+        Assert.assertEquals(Operation.PURCHASE.getOperation(), "p");
+        Assert.assertEquals(Operation.RETURN.getOperation(), "r");
     }
 
     @Test
-    public void testBalanceIsNull_ok() {
+    public void testBalance_ok() {
         expected = new HashMap<>();
-        expected.put(fruit, value);
-        CustomDataBase.storage.put(fruit, value);
-        BALANCE_OPERATION.changeValue(FRUIT_DAO, fruit, value);
-        Integer actual = CustomDataBase.storage.get(fruit);
-        Assert.assertEquals(expected.get(fruit), actual);
+        expected.put(fruitApple, value);
+        expected.put(fruitBanana, value);
+        BALANCE_OPERATION.changeValue(FRUIT_DAO, fruitApple, value);
+        Assert.assertEquals(expected, CustomDataBase.storage);
     }
 
     @Test
-    public void testSupplyIsNull_ok() {
+    public void testSupply_ok() {
         expected = new HashMap<>();
-        expected.put(fruit, value + value);
-        CustomDataBase.storage.put(fruit, value);
-        SUPPLY_OPERATION.changeValue(FRUIT_DAO, fruit, value);
-        Integer actual = CustomDataBase.storage.get(fruit);
-        Assert.assertEquals(expected.get(fruit), actual);
+        expected.put(fruitApple, value + value);
+        expected.put(fruitBanana, value);
+        CustomDataBase.storage.put(fruitApple, value);
```

Comment:
we shouldn't put fruits to our storage directly, because we do it in `SUPPLY_OPERATION.changeValue` 
(check for all methods in this class)

Reply:
No. We should first put in storage, because he is empty. in this case we will throw NullPointerException. 
If we want to change(changeValue) some value from storage, in first step we should already have some value in this. 
Why out storage in each test empty? because we have @After, where we clear out storage.